lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/list.py

```diff
@@ -13,7 +13,7 @@
     from PySide6.QtGui import QPainter
     from PySide6.QtWidgets import QStyleOptionViewItem, QStyledItemDelegate, QListView, QLabel, QVBoxLayout, QHBoxLayout, QWidget, QCheckBox
 except:
-    from PySide2.QtCore import (
+    from PySide2.QtCore import ( # type: ignore
```

Comment:
What are these type: ignore comment?

Reply:
It's so that the editor doesn't complain about the types it doesn't know about